openhcl/openhcl_boot/src/arch/x86_64/hypercall.rs

```diff
@@ -21,8 +21,7 @@ fn write_hypercall_msr(enable: bool) {
         read_msr(hvdef::HV_X64_MSR_HYPERCALL)
     });
 
-    // SAFETY: just getting the address.
-    let hypercall_page_num = unsafe { addr_of!(HYPERCALL_PAGE) } as u64 / HV_PAGE_SIZE;
+    let hypercall_page_num = addr_of!(HYPERCALL_PAGE) as u64 / HV_PAGE_SIZE;
```

Comment:
yay

Reply:
&raw is stable now as well, but I suppose that'll have to land alongside the version update itself

Reply:
I'm surprised there isn't a lint moving away from addr_of to &raw, but we can just do it whenever. Maybe addr_of is going to be deprecated in a release or two or something?

Reply:
1.82 brings other nice things as well, such as `unsafe extern`. Probably the lints requiring their use will come in time.